imap_processing/swe/l1a/swe_l1a.py

```diff
@@ -13,15 +93,23 @@ def swe_l1a(packet_file: str):
 
     Parameters
     ----------
-    packet_file : str
-        The path and filename to the L0 file to read
+    packets: list
+        Decom data list that contains all appIds
     """
-    decom_data = decom_swe.decom_packets(packet_file)
-    logging.info(f"Unpacking data from {packet_file}")
-
-    # If appId is science, then the file should contain all data of science appId
-    if decom_data[0].header["PKT_APID"].raw_value == 1344:
-        logging.info("Processing science data")
-        return swe_science(decom_data=decom_data)
-    else:
-        return decom_data
+    # group data by appId
+    grouped_data = group_by_apid(packets)
```

Comment:
I think you should be able to test this with the current data you've got? Just assert that the group is in sorted order or something else quick.

Reply:
I think there is test for this function. Did you like me to add specific test in there?

Reply:
I think this comment was before you added the test and codecov was failing. Resolving this now, nothing needed here.